packages/core/crates/core-misc/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "core-misc"
+version = "0.1.0"
+authors = ["HOPR Association <tech@hoprnet.org>"]
+edition = "2021"
+description = "Selected core modules"
+homepage = "https://hoprnet.org/"
+repository = "https://github.com/hoprnet/hoprnet"
+license = "LGPL-3.0-only"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+real-base = { path = "../../../real/crates/real-base" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+serde-wasm-bindgen = "0.4"
+wasm-bindgen = "0.2.83"
+wasm-bindgen-test = "0.3.30"
+
+console_error_panic_hook = { version = "0.1.6", optional = true }
+wee_alloc = { version = "0.4.5", optional = true }
```

Comment:
We're currently not using `wee_alloc` anywhere, so better disable it in this crate as well (and comment out the dependency here)